src/config/types.js

```diff
@@ -19,7 +19,7 @@ export type DerivationType = $Values<typeof DERIVATION_TYPES>
 export const WALLET_IMPLEMENTATION_REGISTRY = {
   HASKELL_BYRON: 'haskell-byron', // bip44
   HASKELL_SHELLEY: 'haskell-shelley', // cip1852
-  BYRON: 'byron', // deprecated
+  DAEDALUS_HASKELL_SHELLEY: 'daedalus-haskell-shelley', // cip1852
```

Comment:
I'm not sure if calling the "Daedalus Shelley" makes sense since any wallet can add support for 24.

Reply:
The main reason why I wanted a new wallet ID was basically to have a bit of information on whether a wallet might have been restored from a Daedalus seed (mostly for debugging purposes). But since you mention we may add support for 12 or 18-word or whatever, I reckon it doesn't make much sense to store this information, specially since from the app perspective all wallets correspond to the same implementation. I'll remove that and just keep `HASKELL_SHELLEY`